src/main/java/ithillel/bank/controllers/rest/AccountController.java

```diff
@@ -30,9 +30,9 @@ public AccountDto getAccount(@PathVariable String uid){
 
     @PostMapping("/accounts")
     @Transactional
-    public AccountDto create(@RequestParam String personUid, @RequestParam String iban){
-        log.info("createAccount: personUid={}, iban={}", personUid, iban);
-        return accountService.create(iban, personUid);
+    public AccountDto create(@RequestParam String personUid, @RequestParam String iban, @RequestParam String currency){
```

Comment:
Дуже не звично так (хоча і працює). 
Правильніше створити ще один дто (AccountCreationDto) 
```
public record AccountCreationDto(@JsonProperty("person_uid") String personUid, String iban, String currency){}
```


і тут робити так: 
```suggestion
    public AccountDto create(@RequestBody AccountCreationDto request){
```